### ISSUE TITLE
xournal latex insertion "ok" button greyed out on windows 10 64bit

### ISSUE THREAD

**Issue by 0lm** [xournalpp/xournalpp#3026]
**Affects versions :**
 - OS: Windows 10 64bit
 - Which version of libgtk do you use
     - 3.24.27
 - Version of Xournal++: [e.g. 1.0.16, specific commit, specific branch]
     - 1.1.0+ dev , Git Commit: 1002 c 131
 - Installation method: Windows Exe installer

**Describe the bug**
Latex Insertion "Ok" button is greyed out.

**To Reproduce**
1. freshly install xournal on Windows 10 64bit.
2. install tex live 2021 with the windows installer
3. try to insert latex formula (default example: x^2), but "ok" is greyed out.

**Expected behavior**
Latex Insertion "ok" button not greyed out.

**Additional context**
I found [this](https://github.com/xournalpp/xournalpp/issues/1681) similar issue already. But since I am on Windows and not Linux, I dont really understand the solution of that problem. I recently freshly installed tex live 2021 and I dont know what files should hinder xournal++ from using pdflatex. According to my CMD output, pdflatex successfully installed. And about the icon theme thingy: Since I am on Windows and since xournalpp was freshly installed, I dont think I changed any icon themes. I dont even think that I have different icon themes other than the default ones.

When I go Temp files to check xournalpp error output, I find these logs:

```
This is pdfTeX, Version 3.141592653-2.6-1.40.22 (TeX Live 2021/W32TeX) (preloaded format=pdflatex 2021.4.12)  12 APR 2021 20:50
entering extended mode
 restricted \write18 enabled.
 %&-line parsing enabled.
**C:/Users/USER~1/AppData/Local/Temp/xournalpp-10204/tex/tex.tex

! Emergency stop.
<to be read again> 
                   \protect 
<*> C:/Users/USER~
                    1/AppData/Local/Temp/xournalpp-10204/tex/tex.tex 
Here is how much of TeX's memory you used:
 4 strings out of 478994
 118 string characters out of 5862207
 283038 words of memory out of 5000000
 17591 multiletter control sequences out of 15000+600000
 403430 words of font info for 27 fonts, out of 8000000 for 9000
 1141 hyphenation exceptions out of 8191
 3i,0n,0p,1b,6s stack positions out of 5000i,500n,10000p,200000b,80000s

!  ==> Fatal error occurred, no output PDF file produced!
```
I suppose this is the error code and reason why the "ok" button is greyed out. But ufortunately I dont understand it.

**Comment by personalizedrefrigerator** [comment]
https://github.com/xournalpp/xournalpp/issues/2910 looks similar to this. I don't run Windows, so someone else will have to open a PR!

**Comment by 0lm** [comment]
I see, that resembles my issue. SO the problem is, my long username is shortened to USER~1 and that causes problems for pdflatex? Is there a temporary workaround so it wont shorten my username anymore to make it work properly?

**Comment by mitzal** [comment]
@0lm You can change TEMP environment variable to any existing directory you want (that may break other apps though).
You can open powershell, change `env:TEMP` only for current session and than run xournal.exe from there (this won't affect other apps).

Alternatively, you can try disabling short name generation: [link](https://docs.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2003/cc778996(v=ws.10)?redirectedfrom=MSDN). I haven't test this so not sure if it works.

**Comment by Technius** [comment]
This should've been fixed by #3046.
